src/models/users.js

```diff
@@ -24,6 +24,7 @@ users.virtual('token').get(function () {
 users.virtual('capabilities').get(function () {
   let acl = {
     user: ['read'],
+    writer: ['read', 'create'],
     editor: ['read', 'create', 'update'],
     admin: ['read', 'create', 'update', 'delete'],
   };
```

Comment:
I am not sure why jest can't be installed, I would suggest finding a lab TA and see if it can be sorted out